### PR TITLE
Ignore IUpdateModel and BuildPartDisplayContext in console_log

### LINKED ISSUE #11260: console_log on FlowPart shape returns logs of C# Runtime objects
### To Reproduce
1. Use `{{ Model.Content | console_log }}` or `@Orchard.ConsoleLog((object)Model)`
2. Log model containing `FlowPart`
4. See browser's log  and find C# Runtime properties logged.

### Expected behavior
It should only log Shape related properties.

### PR DESCRIPTION
**Pull request by ns8482e**
Ignore `IUpdateModel` and `BuildPartDisplayContext` properties in console_log for Shape json

Fixes #11260

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Flows/ViewModels/BagPartEditViewModel.cs [hunk 1]
```diff
@@ -16,6 +16,8 @@ public class BagPartEditViewModel
         [BindNever]
         public BagPart BagPart { get; set; }
 
+        [System.Text.Json.Serialization.JsonIgnore]
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Flows/ViewModels/BagPartEditViewModel.cs:19]: Isn't there a common attribute that doesn't require referencing these libraries?

Comment (ns8482e) [reply src/OrchardCore.Modules/OrchardCore.Flows/ViewModels/BagPartEditViewModel.cs:19]: I added as - I guess NewtosoftJson's  `JsonIgnore` wont work for `System.Text.Json`

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Flows/ViewModels/BagPartEditViewModel.cs:19]: `[IgnoreDataMember]`, or `[NonSerialized]` might work for both libraries

Comment (ns8482e) [reply src/OrchardCore.Modules/OrchardCore.Flows/ViewModels/BagPartEditViewModel.cs]: `NonSerialized is not allowed on properties
However `IgnoreDataMember` is working - Updated the PR

**Comment by ns8482e** [comment]
@sebastienros @deanmarcussen   It's be nice to have this included in 1.4

**Review by deanmarcussen — APPROVED** [review]

**Review by Skrypt — APPROVED** [review]

**Pull request merged**